src/TLuaInterpreter.cpp

```diff
@@ -12531,6 +12531,67 @@ int TLuaInterpreter::getCmdLine(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#addCmdLineSuggestion
+int TLuaInterpreter::addCmdLineSuggestion(lua_State* L)
+{
+    int n = lua_gettop(L);
+    QString name = "main";
+
+    if (n > 1) {
+        name = CMDLINE_NAME(L, 1);
+    }
+    if (!lua_isstring(L, n)) {
+        lua_pushfstring(L, "addCmdLineSuggestion: bad argument #%d (suggestion text as string expected, got %s)", n, luaL_typename(L, n));
+        return lua_error(L);
+    }
+    QString text{lua_tostring(L, n)};
+
```

Comment:
I think you can get rid of the separating newlines here

Reply:
Done.